test/Claims.t.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {TokenFixture} from "./utils/TokenFixture.sol";
+import {Claims} from "../src/Claims.sol";
+import {IClaims} from "../src/interfaces/IClaims.sol";
+import {CurrencyLibrary, Currency} from "../src/types/Currency.sol";
+import {MockClaims} from "../src/test/MockClaims.sol";
+
+contract ClaimsTest is TokenFixture, Test {
```

Comment:
For what reason does it inherit from tokenFixture?

Reply:
for currency0 and currency1